__tests__/app.test.js

```diff
@@ -248,10 +248,86 @@ describe("/api/articles/:article_id/comments", () => {
       })
   })
   describe("POST", () => {
+
     test("Status 201 - returns the posted comment", () => {
+      const incomingComment = {
+        username: "icellusedkars",
+        body: "Posting my first comment!"
+    }
+      return request(app)
+      .post('/api/articles/2/comments')
+      .send(incomingComment)
+      .expect(201)
+      .then(({ body}) => {
+          expect(body).toBeInstanceOf(Object);
+          expect(body).toHaveProperty('postedComment');
+          expect(body.postedComment).toEqual(
```

Comment:
I'd send the data back on the key of `comment` to keep your response keys uniform (we don't want to see `newComment` or `postedComment` keys in our API responses)